modules/custom/az_paragraphs/az_paragraphs_text_media/az_paragraphs_text_media.module

```diff
@@ -8,7 +8,7 @@
 /**
  * Implements hook_theme().
  */
-function az_paragraphs_text_media_theme($existing, $type, $theme, $path) {
+function az_paragraphs_text_media_theme($existing, $type, $theme, $path): array {
```

Comment:
FWIW, I think it'd probably be better to add return type hints to all functions/methods in a separate issue/PR, maybe as part of preparing for Drupal 9.3.x/10.0.x or PHP 8 but maybe that's just me.

Reply:
```suggestion
function az_paragraphs_text_media_theme($existing, $type, $theme, $path) {
```